lbry/schema/mime_types.py

```diff
@@ -166,10 +168,41 @@
     '.wmv': ('video/x-ms-wmv', 'video')
 }
 
+# maps detected extensions to the possible analogs
+# i.e. .cbz file is actually a .zip
+synonyms_map = {
+    '.zip': ['.cbz'],
+    '.rar': ['.cbr'],
+    '.ar': ['.a']
+}
+
+log = logging.getLogger(__name__)
+
 
 def guess_media_type(path):
     _, ext = os.path.splitext(path)
     extension = ext.strip().lower()
+
+    # try detecting real file format if path points to a readable file
+    try:
+        kind = filetype.guess(path)
+        if kind:
+            realext = f".{kind.extension}"
```

Comment:
In Python the standard variable naming convention is to use full words and to put underscores between words.

Reply:
Adjusted.